Directory.Packages.props

```diff
@@ -139,6 +139,7 @@
     <PackageVersion Include="OpenTelemetry.Instrumentation.Http" Version="1.9.0" />
     <PackageVersion Include="OpenTelemetry.Instrumentation.AspNetCore" Version="1.9.0" />
     <PackageVersion Include="OpenTelemetry.Instrumentation.Runtime" Version="1.9.0" />
+    <PackageVersion Include="OpenTelemetry.Instrumentation.ConfluentKafka" Version="0.1.0-alpha.2" />
```

Comment:
What's the plan for shipping this library stable? We won't be able to ship `Aspire.Confluent.Kafka` stable again until this package goes stable.

Reply:
https://github.com/open-telemetry/semantic-conventions/blob/main/docs/messaging/messaging-spans.md and https://github.com/open-telemetry/semantic-conventions/blob/main/docs/messaging/messaging-metrics.md needs to be stable first. Than we can think about making this package stable.

Reply:
I'm assuming that isn't going to happen this calendar year, is that correct?

If that is the case, maybe we need to think about "vendoring" this code and compiling it into the Aspire.Confluent.Kafka library like we do with SqlServer and Redis here:

https://github.com/dotnet/aspire/tree/main/src/Vendoring

Reply:
So we're going to need to copy the implementation @eerhardt ?

Reply:
I do not found any ETA for this. My feeling is more likely - early next year.

Reply:
> So we're going to need to copy the implementation @eerhardt ?

If we want to continue shipping "stable" versions of the Kafka client integration library.